components/Dialogs/tests/unit/MDCAlertControllerTests.m

```diff
@@ -237,4 +241,38 @@ - (void)testTheViewIsNotLoadedWhenPropertiesAreSet {
   XCTAssertFalse(alert.isViewLoaded);
 }
 
+- (void)testAccessibilityIdentifiersAppliesToAlertControllerViewButtons {
+  // Given
+  MDCAlertController *alertController = [MDCAlertController alertControllerWithTitle:@"Title"
+                                                                             message:@"message"];
+  MDCAlertAction *action1 = [MDCAlertAction actionWithTitle:@"button1" handler:nil];
+  action1.accessibilityIdentifier = @"1";
+  MDCAlertAction *action2 = [MDCAlertAction actionWithTitle:@"buttonA" handler:nil];
+  action2.accessibilityIdentifier = @"A";
+
+  // When
+  [alertController addAction:action1];
+  [alertController addAction:action2];
+  
+  // Force the view to load
+  if (@available(iOS 9.0, *)) {
+    [alertController loadViewIfNeeded];
+  } else {
+    (void)alertController.view;
+  }
+
+  // Then
+  NSArray<UIButton *> *buttons = alertController.alertView.actionButtons;
+  XCTAssertEqual(buttons.count, 2U);
```

Comment:
What is the `2U`?

Reply:
2 as an unsigned value.

Reply:
Not quite. Since `count` is an NSUInteger, when comparing it to another value (a constant in this case) the comparison should be of the same type. Some build systems will complain if a signed value and unsigned value are compared, so it's typically preferred to just explicitly mark that this is the unsigned value 2 and not the signed value 2.